cmd/destroy/destroy.go

```diff
@@ -178,6 +186,14 @@ func Destroy(ctx context.Context) *cobra.Command {
 			}
 			os.Setenv("KUBECONFIG", kubeconfigPath)
 			defer os.Remove(kubeconfigPath)
+
+			// when option --all the cmd will destroy everything at the namespace and return
+			if options.DestroyAll {
+				if !okteto.Context().IsOkteto {
+					return errors.New("option `--all` is not available for non-Okteto clusters. Learn more: https://www.okteto.com/docs/self-hosted/")
+				}
+				return c.runDestroyAll(ctx, options)
```

Comment:
We are losing the `TrackDestroy`, @pchico83 is it needed to track this action? Should we include some data in the tracking to indicate that the new flag was being used?

Reply:
is we keep the same event for both destroys we wont see the difference of using `--all` flag, i think we either add more meta to this event or we create a new event for destroyall. WDYT?

Reply:
let's add an additional field to the `destroy` event. It's what we do for other commands